lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -103,7 +103,12 @@ namespace {
     ///
     /// \param foundInType The type through which we found the
     /// declaration.
-    void add(ValueDecl *found, ValueDecl *base, Type foundInType) {
+    ///
+    /// \param promotedInstanceRef true if the lookup result to be added was
+    /// actually looked up on an instance but promted to a type to look up an
```

Comment:
Typo - "promted"

I would just name this "fixItAddLeadingDot" or something really explicit.